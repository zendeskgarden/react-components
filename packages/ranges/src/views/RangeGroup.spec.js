/**
 * Copyright Zendesk, Inc.
 *
 * Use of this source code is governed under the Apache License, Version 2.0
 * found at http://www.apache.org/licenses/LICENSE-2.0.
 */

import React from 'react';
import { render, renderRtl } from 'garden-test-utils';

import RangeGroup from './RangeGroup';

describe('RangeGroup', () => {
  it('renders default styling', () => {
    const { container } = render(<RangeGroup />);

    expect(container.firstChild).toHaveClass('c-range');
  });

  it('renders RTL styling', () => {
    const { container } = renderRtl(<RangeGroup />);

    expect(container.firstChild).toHaveClass('is-rtl');
  });

  it('renders inline styling if provided', () => {
    const { container } = render(<RangeGroup inline />);

    expect(container.firstChild).toHaveClass('c-range--inline');
  });
});
