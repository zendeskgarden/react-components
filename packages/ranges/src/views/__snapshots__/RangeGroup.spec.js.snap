// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RangeGroup renders RTL styling 1`] = `
<div
  className="c-range is-rtl "
  data-garden-id="ranges.range_group"
  data-garden-version="version"
/>
`;

exports[`RangeGroup renders default styling 1`] = `
<div
  className="c-range "
  data-garden-id="ranges.range_group"
  data-garden-version="version"
/>
`;

exports[`RangeGroup renders inline styling if provided 1`] = `
<div
  className="c-range c-range--inline "
  data-garden-id="ranges.range_group"
  data-garden-version="version"
/>
`;
